main.py

```diff
@@ -1,34 +1,42 @@
+token = '' # Input your token here.
+
+
 # Imports
 import discord
 import asyncio
 import random
 import re
 
+
+
 # From imports
 from discord.ext import commands
 from datetime import datetime, timedelta
 from colorama import Fore, Style
 
-token = ''  # Input your token here.
+
+# Error Imports
+from discord.ext.commands import MissingPermissions
+
+
+
 
 intents = discord.Intents.default()
-bot = commands.Bot(command_prefix='!', case_insensitive=True, intents=intents)
+bot = commands.Bot(command_prefix = '!' , case_insensitive = True , intents = intents)
 bot.remove_command('help')
 
 
 # Commands
 
 # Giveaway Command
-@bot.command(aliases=['start', 'g'])
-@commands.has_permissions(manage_guild=True)
+@bot.command(aliases = ['start , g'])
+@commands.has_permissions(manage_guild = True)

```

Comment:
Why does a bot need this permissions?

Reply:
The bot doesn't it's for the user, if the user has the permission, they can use the command otherwise it returns an error. :/

Reply:
right, sry.

Reply:
so this is no longer an error